docs/blog/rendered/ibis-version-6.1.0-release.ipynb

```diff
@@ -411,7 +411,7 @@
    "source": [
```

Comment:
<p>There are extra backticks in the code box here, and the first word of the second sentence is missing an <code>r</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6769/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>should be fixed </p>